src/components/verification-grid/verification-grid.ts

```diff
@@ -97,22 +105,23 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
   @property({ attribute: "get-page", type: Function, converter: callbackConverter as any })
   public getPage?: PageFetcher;
 
-  /** Automatically advance to the next page after a decision is made */
-  @property({ attribute: "auto-page", type: Boolean, converter: booleanConverter })
-  public autoPage = true;
-
-  /** Pre-fetch the next page of recordings */
-  @property({ attribute: "pre-fetch", type: Boolean, converter: booleanConverter })
-  public preFetch = true;
-
   @state()
   private spectrogramElements?: TemplateResult<1> | TemplateResult<1>[];
 
   @state()
   private historyHead = 0;
 
+  /** selector for oe-decision elements */
   @queryAllDeeplyAssignedElements({ selector: "oe-decision" })
-  private decisionElements?: DecisionComponent[];
+  private oeDecisionElements!: DecisionComponent[];
+
+  /** selector for oe-verification elements */
+  @queryAllDeeplyAssignedElements({ selector: "oe-verification" })
+  private verificationDecisionElements!: VerificationComponent[];
+
+  /** selector for oe-classification elements */
+  @queryAllDeeplyAssignedElements({ selector: "oe-classification" })
```

Comment:
This was a conscious design decision because slots are user defined.

In my opinion, users should have the ability to (but shouldn't) create a document template like

```html
<oe-verification-grid>
	<template>
		<!-- snipped -->
	</template>

	<nav class="sidebar">
		<oe-classification>Classification in side bar</oe-classification>
	</nav>
</oe-verification-grid>
```

Reply:
Not that I want the use of `slot` encouraged here either, but isn't that the role that `slot` should be filling?

```
<oe-verification-grid>
	<template>
		<!-- snipped -->
	</template>

	<nav slot="classification" class="sidebar">
		<oe-classification>
			<p>Classification in side bar</p>
			<p>That is also not in the default slots top level</p>
		</oe-classification>
	</nav>
</oe-verification-grid>
```

Regardless, I don't think we need this level of customization. If the `<oe-classification/verification>` element is not the first child it is configured incorrectly.

If we ever need more customization here - which seems unlikely given the constraints of the UI - we make this more flexible later.